src/app/components/views/weather-view/weather-view.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from '@angular/core';
+
+import { WeatherAlertResponse } from 'src/app/models/weather-alert.model';
+import { WeatherService } from 'src/app/services/weather.service';
+
+@Component({
+  selector: 'app-weather-view',
+  templateUrl: './weather-view.component.html',
+  styleUrls: ['./weather-view.component.scss']
+})
+export class WeatherViewComponent implements OnInit {
+  public weatherAlertResponse: WeatherAlertResponse;
+
+  constructor(private weatherService: WeatherService) { }
+
+  ngOnInit(): void {
+
+    this.wsGetResults();
+
+}
+
+/**
+ * @description- looks like black magic to me
+ * @returns- something Micah coded, probably.
+ */
+
+  private wsGetResults(): any {
```

Comment:
return statement for method says return type of `any`. what is this method returning?

also, please update the method name. `ws` doesnt tell me anything....you want a name to be descriptive as a jsdoc. `subscribeToWeatherService()`

Reply:
put this in as a joke/placeholder while I was away